### PR TITLE
Jm/omoo test

### PR DESCRIPTION
**Pull request by tarekelgindy**

### CODE REVIEW

File: LocalFeeder/FeederSimulator.py [hunk 1]
```diff
@@ -708,6 +708,46 @@ def set_properties_to_inverter(self, inverter: str, inv_control: InverterControl
...
+        pmpp = None
+        while True:
+            if dss.PVsystems.Name() == pv_system:
+                print(dss.PVsystems.Name())
+                irradiance = dss.PVsystems.Irradiance()
+                pmpp = dss.PVsystems.Pmpp()
+            if not dss.PVsystems.Next() > 0:
+                break
+        if irradiance is None or pmpp is None:
+            import pdb;pdb.set_trace()
```
Comment (josephmckinsey) [inline LocalFeeder/FeederSimulator.py:741]: This is probably not needed. P.S. Use --pdb in pytest to automatically enter debug mode on failure.

File: LocalFeeder/FeederSimulator.py [hunk 2]
```diff
@@ -708,6 +708,46 @@ def set_properties_to_inverter(self, inverter: str, inv_control: InverterControl
...
+            raise ValueError(f"Irradiance or PMPP not found for {pv_system}")
+        return irradiance*pmpp
+
+
+
```
Comment (josephmckinsey) [inline LocalFeeder/FeederSimulator.py:746]: Nit: maybe run a formatter to remove these lines

File: LocalFeeder/tests/test_feeder.py [hunk 3]
```diff
@@ -592,3 +592,35 @@ def test_inv_combined_control(federate_config):
...
+        [
+            FeederSimulator.Command(
+                obj_name="PVSystem.113",
+                obj_property="irradiance",
+                val=0.2, 
+            )
+        ]
+    )
+    sim.set_pv_output("113",20,5)
+
```
Comment (josephmckinsey) [inline LocalFeeder/tests/test_feeder.py:625]: I think we talked about this, but could we check that the pv is actually set there using a solve + getting the power injections.

**Review by josephmckinsey — CHANGES_REQUESTED** [review]

File: LocalFeeder/FeederSimulator.py [hunk 4]
```diff
@@ -708,6 +708,55 @@ def set_properties_to_inverter(self, inverter: str, inv_control: InverterControl
...
+        command = [Command(obj_name=obj_name,obj_property="%Pmpp",val=str(obj_val)), Command(obj_name=obj_name,obj_property="kvar",val=str(q)), Command(obj_name=obj_name,obj_property="%Cutout", val="0"),  Command(obj_name=obj_name,obj_property="%Cutin", val="0")]
+        self.change_obj(command)
+        
+    def get_pv_output(self,pv_system):
+        dss.PVsystems.First()
+        irradiance = None
+        pmpp = None
+        while True:
+            if dss.PVsystems.Name() == pv_system:
+                print(dss.PVsystems.Name())
```
Comment (josephmckinsey) [inline LocalFeeder/FeederSimulator.py:738]: Do we want to keep this print?

File: LocalFeeder/FeederSimulator.py [hunk 5]
```diff
@@ -708,6 +708,55 @@ def set_properties_to_inverter(self, inverter: str, inv_control: InverterControl
...
+                break
+        return kw,kvar
+
+    def get_max_pv_available(self,pv_system):
+        dss.PVsystems.First()
+        irradiance = None
+        pmpp = None
+        while True:
+            if dss.PVsystems.Name() == pv_system:
+                print(dss.PVsystems.Name())
```
Comment (josephmckinsey) [inline LocalFeeder/FeederSimulator.py:751]: print again?

**Review by josephmckinsey — APPROVED** [review]
Might want to run black or ruff on FeederSimulator.py

File: LocalFeeder/FeederSimulator.py (hunk 4 above)
Comment (tarekelgindy) [reply LocalFeeder/FeederSimulator.py:738]: Ok I'll get rid of these.

**Pull request merged**